components/CookieConsentConfig.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @type {UserConfig}
+ */
+const pluginConfig = {
+  current_lang: "en",
+  autoclear_cookies: true, // default: false
+  page_scripts: true, // default: false
+  test: "ciao",
+  force_consent: true, // default: false
+  // mode: 'opt-in'                          // default: 'opt-in'; value: 'opt-in' or 'opt-out'
+  // delay: 0,                               // default: 0
+  // auto_language: null                     // default: null; could also be 'browser' or 'document'
+  // autorun: true,                          // default: true
+  // hide_from_bots: false,                  // default: false
+  // remove_cookie_tables: false             // default: false
+  // cookie_name: 'cc_cookie',               // default: 'cc_cookie'
+  // cookie_expiration: 182,                 // default: 182 (days)
+  // cookie_necessary_only_expiration: 182   // default: disabled
+  // cookie_domain: location.hostname,       // default: current domain
+  // cookie_path: '/',                       // default: root
+  // cookie_same_site: 'Lax',                // default: 'Lax'
+  // use_rfc_cookie: false,                  // default: false
+  // revision: 0,                            // default: 0
+
+  onFirstAction: function (user_preferences, cookie) {
+    // callback triggered only once
+  },
+
+  onAccept: function (cookie) {
+    // ...
+  },
+
+  onChange: function (cookie, changed_preferences) {
+    // ...
```

Comment:
Nothing happens when someone change consents to the cookies!

Reply:
This function is for some cleanup logic implementations like disabling the GA4 scripts (which is only valid if we add the user's functionality to change the consent after being accepted once as this function will be executed only if consent has already been given). The consent content is changing.

![Screenshot (294)](https://user-images.githubusercontent.com/76250685/211202818-0a470fef-77f8-4ec5-a550-00d5185930bd.png)

Pls, let me know if I am missing anything 😓